subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/GitAttributesGenerator.java

```diff
@@ -33,7 +33,10 @@ public void generate(InitSettings settings) {
                 writer.println("#");
                 writer.println("# https://help.github.com/articles/dealing-with-line-endings/");
                 writer.println("#");
-                writer.println("# These are explicitly windows files and should use crlf");
+                writer.println("# Linux start script should use lf");
+                writer.println("/gradlew        text eol=lf");
+                writer.println();
+                writer.println("# These are Windows script files and should use crlf");
                 writer.println("*.bat           text eol=crlf");
```

Comment:
Would it make sense to make this more specific too? It appears the intention of #9583 was to only cover `gradlew.bat`; if users want this for all Windows script files, then they should probably manually specify this, but it shouldn't be Gradle who is doing this.
Maybe there are also other (non-text) files with extension `.bat` for which this entry would be causing issues.
(Though it is rather unlikely and there have no been any issues for this.)

```suggestion
                writer.println("# Windows start script should use crlf");
                writer.println("/gradlew.bat      text eol=crlf");
```

Reply:
This is like that because the `application` plugin can generate `.bat` files.